common/eth2_network_config/src/lib.rs

```diff
@@ -154,6 +154,26 @@ impl Eth2NetworkConfig {
         }
     }
 
+    pub fn genesis_state_root<E: EthSpec>(&self) -> Result<Option<Hash256>, String> {
```

Comment:
Do we need the `Option` here? 
Might be able to simplify quite a few functions if we remove this `Option`

Reply:
Hmm, I think we need the `None` case for networks with unknown genesis. I'll refactor this so that it returns `None` rather than an error in that case.